.github/workflows/restart_recovery_suite.yml

```diff
@@ -0,0 +1,118 @@
+name: Restart recovery suite
+on:
+  schedule:
+    - cron: '0 5 * * *'
+  push:
+    branches:
+      - '*' # matches every branch
+  pull_request:
+    branches:
+      - '*' # matches every branch
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    timeout-minutes: 600
+    runs-on: ubuntu-18.04
+    outputs:
+      err_output: ${{ steps.prepare_err_artifacts.outputs.test}}
+    strategy:
+      fail-fast: false
+      matrix:
+            compiler:
+                - "CONCORD_BFT_CONTAINER_CC=clang CONCORD_BFT_CONTAINER_CXX=clang++"
+            ci_build_type:
+                - "-DCMAKE_BUILD_TYPE=RELEASE -DBUILD_COMM_TCP_TLS=FALSE"
+                - "-DCMAKE_BUILD_TYPE=RELEASE -DBUILD_COMM_TCP_TLS=TRUE"
+            use_s3_obj_store:
+                - "-DUSE_S3_OBJECT_STORE=ON"
+                - "-DUSE_S3_OBJECT_STORE=OFF"
```

Comment:
Let's simplify the CI test matrix to only test with TLS turned off, and S3 store turned off. 
@titusen and @HristoStaykov . what do you think?

Reply:
I agree, the tests we currently include focus mostly on corner cases and restart recovery that are independent of the transport layer used. I think also S3 is only used in RO replicas, which we don't have currently in the test suite.

Reply:
Done